users/users.go

```diff
@@ -65,10 +66,13 @@ type UserRepository interface {
 	RetrieveByID(ctx context.Context, id string) (User, error)
 
 	// RetrieveAll retrieves all users for given array of userIDs.
-	RetrieveAll(ctx context.Context, offset, limit uint64, userIDs []string, email string, m Metadata) (UserPage, error)
+	RetrieveAll(ctx context.Context, status string, offset, limit uint64, userIDs []string, email string, m Metadata) (UserPage, error)
 
 	// UpdatePassword updates password for user with given email
 	UpdatePassword(ctx context.Context, email, password string) error
+
+	// ChangeStatus changes users status to enabled or disabled
+	ChangeStatus(ctx context.Context, u User, status string) error
```

Comment:
While this is practically user update, no need to pass the entire user - the ID is sufficient.